conf/base/bblayers.conf

```diff
@@ -3,22 +3,24 @@
 POKY_BBLAYERS_CONF_VERSION = "2"
 
 BBPATH = "${TOPDIR}"
+BSPDIR := "${@os.path.abspath(os.path.dirname(d.getVar('FILE', True)) + '/../..')}"
+
 BBFILES ?= ""
 
 BBLAYERS ?= " \
-  ##OEROOT##/meta \
-  ##OEROOT##/meta-poky \
-  ##OEROOT##/meta-yocto-bsp \
-  ##OEROOT##/meta-qt4 \
-  ##OEROOT##/meta-qt5 \
-  ##OEROOT##/meta-uav \
-  ##OEROOT##/meta-ros \
-  ##OEROOT##/meta-intel-aero \
-  ##OEROOT##/meta-intel-aero-connectivity \
-  ##OEROOT##/meta-intel-realsense \
-  ##OEROOT##/intel-aero-samples \
-  ##OEROOT##/meta-openembedded/meta-oe \
-  ##OEROOT##/meta-openembedded/meta-python \
-  ##OEROOT##/meta-openembedded/meta-networking \
-  "
+  ${BSPDIR}/sources/poky/meta \
```

Comment:
I guess I could include that rationale on the commit message too. I feel that the changes in the manifest file are minor to allow a sources directory and the benefits are substantial to devs that also contribute to the poky project. I'm very open to feedback though, What do you think @lucasdemarchi ?